### ISSUE TITLE
Only works on main screen

### ISSUE THREAD

**Issue by Bogoth** [pbxqdown/gnome-shell-extension-transparent-window#4]
### System information
- **OS Platform and Distribution (e.g., Linux Ubuntu 18.04.3)**:
- **GNOME version 3.28.2**:
- **Exact command to reproduce**:

You can obtain the GNOME version by:
1. Open the Activities overview and start typing About.
2. Click on About to open the panel. A window appears showing information about your system, including your distribution's name and the GNOME version.

### Describe the problem
This extension only seems to work on the main display.  If I change my main display it will change which screen is able to make things transparent.

### Source code / logs 
Include any logs or source code that would be helpful to diagnose the problem.   
First, in the preference, Set the log verbose level to "Debug";  
Then you can view gnome shell extension's log via:  
```
journalctl /usr/bin/gnome-shell -r |grep TransparentWindow
```
To view logs dynamically:
```
journalctl /usr/bin/gnome-shell -f |grep TransparentWindow
```
No relevant logs.

**Comment by pbxqdown** [comment]
Hi Bogoth, thanks for the suggestion. There is a todo in the code: https://github.com/pbxqdown/gnome-shell-extension-transparent-window/blob/master/extension.js#L95. Please let me know if you have any clue/suggestion/pull-request on the implementation. Thanks.

**Comment by gha3mi** [comment]
Hi, I have the same issue on Ubuntu 22.04.1 GNOME 42.4.

**Comment by pbxqdown** [comment]
Fixed by #24
